ci/github-script/labels.js

```diff
@@ -308,6 +313,7 @@ module.exports = async function ({ github, context, core, dry }) {
             'is:open',
             `updated:>=${cutoff.toISOString()}`,
           ].join(' AND '),
+          per_page: 100,
```

Comment:
What's the max? IIRC 250?

What's your thought process behind choosing 100 specifically?